assembly/std/storage.ts

```diff
@@ -1,12 +1,64 @@
+/**
+ * This module contains functions for interacting with the key-value datastore, which is used for persistent storage of
+ * data on the blockchain.
+ *
+ * The supported value types for the datastore include `string`, `Args` and `StaticArray<u8>`.
+ *
+ * This module also provides functions for setting the executable bytecode of a smart contract address.
+ *
+ * @remarks
+ * The datastore is used to store data that is expected to persist between contract executions, such as contract
+ * state or user information.
+ *
+ * The [set](../functions/Storage.set.html), [get](../functions/Storage.get.html),
```

Comment:
You can use @link 

like here https://github.com/massalabs/as/blob/main/packages/as-types/assembly/serialization/arrays.ts#L13

see doc here https://typedoc.org/tags/link/

try something like this:

```typescript
 * The {@link set}, {@link get},
```

Reply:
A lot cleaner thanks